doc/classes/Node3D.xml

```diff
@@ -224,143 +231,155 @@
 				[b]Note:[/b] The gizmo object would typically be an instance of [EditorNode3DGizmo], but the argument type is kept generic to avoid creating a dependency on editor classes in [Node3D].
 			</description>
 		</method>
 		<method name="show">
 			<return type="void" />
 			<description>
-				Enables rendering of this node. Changes [member visible] to [code]true[/code].
+				Allows this node to be rendered. Equivalent to setting [member visible] to [code]true[/code]. This is the opposite of [method hide].
 			</description>
 		</method>
 		<method name="to_global" qualifiers="const">
 			<return type="Vector3" />
 			<param index="0" name="local_point" type="Vector3" />
 			<description>
-				Transforms [param local_point] from this node's local space to world space.
+				Returns the [param local_point] converted from this node's local space to global space. This is the opposite of [method to_local].
 			</description>
 		</method>
 		<method name="to_local" qualifiers="const">
 			<return type="Vector3" />
 			<param index="0" name="global_point" type="Vector3" />
 			<description>
-				Transforms [param global_point] from world space to this node's local space.
+				Returns the [param global_point] converted from global space to this node's local space. This is the opposite of [method to_global].
 			</description>
 		</method>
 		<method name="translate">
 			<return type="void" />
 			<param index="0" name="offset" type="Vector3" />
 			<description>
-				Changes the node's position by the given offset [Vector3].
-				Note that the translation [param offset] is affected by the node's scale, so if scaled by e.g. [code](10, 1, 1)[/code], a translation by an offset of [code](2, 0, 0)[/code] would actually add 20 ([code]2 * 10[/code]) to the X coordinate.
+				Adds the given translation [param offset] to the node's [member position] in local space. Unlike [method translate_object_local], the final translation is multiplied by the node's [member scale].
 			</description>
 		</method>
 		<method name="translate_object_local">
 			<return type="void" />
 			<param index="0" name="offset" type="Vector3" />
 			<description>
-				Changes the node's position by the given offset [Vector3] in local space.
+				Adds the given translation [param offset] to the node's [member position] in local space.
 			</description>
 		</method>
 		<method name="update_gizmos">
 			<return type="void" />
 			<description>
-				Updates all the [Node3D] gizmos attached to this node.
+				Updates all the [EditorNode3DGizmo] objects attached to this node. Only works in the editor.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="basis" type="Basis" setter="set_basis" getter="get_basis">
-			Basis of the [member transform] property. Represents the rotation, scale, and shear of this node.
+			Basis of the [member transform] property. Represents the rotation, scale, and shear of this node in local space (relative to the parent node).
```

Comment:
Node3D in its own local/object coordinate system is always not rotated/scaled, its Basis would be `Basis.IDENTITY`. This `basis` is in parent space / transforms from local to parent space.